docs/arcadia-users-group/20240723-design-fundamentals/lesson.md

```diff
@@ -0,0 +1,240 @@
+- *Explanatory* visualizations highlight the key findings, helping viewers understand cause and effect, trends, and conclusions. This can be achieved through data selection, annotations, direct labels, highlighting, and encoding emphasis.  
+- *Exploratory* visualizations allow viewers to interact with a dataset to draw conclusions themselves. This can be achieved through interactivity, encoding categories with equal visual weight.
+- What kind of experience are you looking to create?
+
+
+#### 3. What kind of media will you be using?
+Before starting on a design, consider through what medium it will be seen. Different media have different constraints and affordances that will affect how you design your figure.  For example:
+- **Pub:** This medium can handle more detail/dense information, text, and annotations.
+- **Twitter / X / Bluesky / etc.**: On social media platforms, effective posts are image heavy with sparse text. You may want to consider a using Graphical Abstract (GA) or animation to draw a reader's attention.
+- **Slide:** During a talk, information can be broken up/introduced across multiple slides. You can also use animations to guide the viewer’s attention.
+- **Poster:** With a large, printed medium such as a poster, the grouping of information and visual flow becomes very crucial. Legibility at a distance is also important.
+
+
+#### 4. What format will you use?
+- **Data figures:** Figures that show raw data, relationships, or trends. These are the most common type of scientific figure.
+- **Methods figures:** Figures that illustrate a process or technique.
+- **Graphical Abstracts and Illustrations:** These are used to summarize a paper or concept in a single image. They can be useful for showing:
+  - Dense explanatory info
+  - A more complex/complete story
+  - A step-by-step guide
+
+
+## Layout
+Once we have our key takeaways, we know who we're designing for, where it's going, and what it is, we can start the actual design process.  At a high level, how we arrange and compose figures impacts their comprehensibility. We'll discuss principles of layout that you can use to guide the viewer through your figure.  Let's discuss the tools we have for arranging and composing figures.
+
+### Gestalt Principles
+The term “gestalt” refers to the idea that the whole is greater than the sum of its parts.  The gestalt principles of design are a set of rules that describe how we perceive visual elements as a whole. 
+
+For example, consider the image below. What do you see?
+![Emergence example](emergence_example.png)
+
+You might say that the image is a of a dalmatian. But if you look closer, you’ll see that the image is actually made up of a series of black and white shapes.  This is an example of the gestalt principle of *emergence*, which describes how we perceive a whole image before we see its individual parts.
```

Comment:
We don't need to double space between sentences; that syntax is a holdover from conventions used with typewriters. We should remove double-spaces throughout.

```suggestion
You might say that the image is a of a dalmatian. But if you look closer, you’ll see that the image is actually made up of a series of black and white shapes. This is an example of the gestalt principle of *emergence*, which describes how we perceive a whole image before we see its individual parts.
```